.circleci/Makefile

```
@@ -54,7 +56,7 @@ $(SOURCE_DIR):
 $(shell [ -d .tmp ] || mkdir .tmp)
 
 .PHONY: $(CONFIG)
-$(CONFIG): $(OUT)
+$(CONFIG): $(OUT) $(GO_VERSION_FILE )
```

Looks like an extra space snuck in here:
```suggestion
$(CONFIG): $(OUT) $(GO_VERSION_FILE)
```